styles.css

```
@@ -7721,6 +7721,149 @@ body:has(#side-panel.active) #close-fullscreen-cover-btn {
+    position: relative;
+    height: 160px;
+    width: 28px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.graphic-eq-band-slider-wrap input[type='range'] {
+    writing-mode: vertical-lr;
+    direction: rtl;
+    appearance: slider-vertical;
```

Stylelint (17.6.0)

[error] Unexpected unknown value "slider-vertical" for property "appearance" (declaration-property-value-no-unknown)

(declaration-property-value-no-unknown)